### ISSUE TITLE
Feature: support set list style type by insertList function

### ISSUE THREAD

**Issue by milkrong** [facebook/lexical#6327]
I am woking on the List plugin. While I have to set the list style type by using theme.
Is there any way to support user to change it?  as we can set list style type in css
<img width="338" alt="image" src="https://github.com/facebook/lexical/assets/15906981/5e69deb9-7833-41f6-911e-e1d42e7fa33e">

**Comment by milkrong** [comment]
> You can set this via the editor theme
> 
> ```
> import type { InitialConfigType } from "@lexical/react/LexicalComposer";
> 
> const themeClass: InitialConfigType["theme"] = {
> 	list: {
> 		ol: "list-decimal",
> 		ul: "list-disc",
> 	}
> };
> ```
> 
> and then pass the `themeClass` into the initial Config of the `LexicalComposer`

@ElasticBottle No, it's not my expected. I want to support set the this type by some selection component in toolbar. No a preset with devleoper

**Comment by Kingscliq** [comment]
Hi @milkrong  has this been fixed?

**Comment by etrepum** [comment]
No, there are only three list types supported by the list node and no support for adding extra metadata. Like most use cases that lexical doesn't support directly, the solution is to subclass the node and use node replacement to add support for additional properties and classes.

**Comment by Kingscliq** [comment]
can I work on it @etrepum ?
